<launch>

    <param name="use_sim_time" value="false"/>
    
    <!-- <include file="$(find velodyne_pointcloud)/launch/VLP16_points.launch" /> -->

    <node pkg="rosbag" type="play" name="rosbag_play" required="true"
         args="-l /home/khk1262/catkin_ws/src/velodyne_filter/resources/2020-07-31-15-43-51.bag"/>


    <node pkg="tf" type="static_transform_publisher" name="map_to_velodyne" output="screen" args="0 0 1 0 0 0 map velodyne 20"/>

    <node pkg="velodyne_filter" type="velodyne_cluster" name="velodyne_cluster_node" >
        <param name="x_axis_min" value="-0.0"/>
        <param name="x_axis_max" value="10.0"/>
        <param name="y_axis_min" value="-10.0"/>
        <param name="y_axis_max" value="10.0"/>
        <param name="z_axis_min" value="-1.0"/>
        <param name="z_axis_max" value="10.0"/>

        <param name="z_axis_angle_" value="0.0"/>
    </node>

    <!-- <node pkg="rviz" type="rviz" name="rviz" output="screen" args="-d $(find velodyne_filter)/resources/cluster.rviz"/> -->

</launch>
